src/js/components/covid19/PublicLawPicker.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Picker } from 'data-transparency-ui';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import GlossaryLink from '../../components/sharedComponents/GlossaryLink';
+
+const options = [
+    {
+        title: 'All COVID-19 Funding Laws', description: 'Data from all related DEFC', value: 'all', link: 'disaster-emergency-fund-code-defc'
+    },
+    { title: 'American Rescue Plan Act of 2021', description: 'Emergency PublicLaw 117-7 (DEFC V)', value: 'american-rescue-plan' },
+    { title: 'Learn more about filtering USAspending data by Public Law in our Data Sources & Methodology page.', value: 'dsm', className: 'ds-and-m' }
+];
+
+const PublicLawPickerOption = ({ title, description, link }) => (
+    <li>
+        <p>{title}</p>
+        <p>
+            {description}
+            {link && (
+                <GlossaryLink term={link} currentUrl="disaster/covid-19" />
```

Comment:
In sandbox the glossary link isn't working for me, but that might just be because it doesn't have the changes we made in sprint 132 for handling query params + glossary links